src/data-loader.ts

```
@@ -663,41 +689,36 @@ export async function loadDailyUsageData(
-			if (Result.isFailure(parseResult)) {
-				// Skip invalid JSON lines
-				continue;
-			}
-
-			const result = usageDataSchema.safeParse(parseResult.value);
-			if (!result.success) {
-				continue;
-			}
-			const data = result.data;
+			try {
+				const parsed = JSON.parse(line) as unknown;
```

ESLint

[error] Unsafe argument of type error typed assigned to a parameter of type `string`.

(ts/no-unsafe-argument)